ACE/ace/config-macosx-mojave.h

```diff
@@ -1,6 +1,8 @@
 #ifndef ACE_CONFIG_MACOSX_MOJAVE_H
 #define ACE_CONFIG_MACOSX_MOJAVE_H
 
+#define __APPLE_USE_RFC_3542
```

Comment:
should this be guarded with an `ifdef` in case its already defined?

Reply:
and some docu maybe why this define is set?